qiskit/visualization/circuit/text.py

```diff
@@ -1057,14 +1054,14 @@ def normalize_width(layer):
             node.layer_width = longest
 
     @staticmethod
-    def controlled_wires(node, layer):
+    def controlled_wires(node, wire_map):
         """
         Analyzes the node in the layer and checks if the controlled arguments are in
         the box or out of the box.
 
         Args:
             node (DAGNode): node to analyse
-            layer (Layer): The layer in which the node is inserted.
+            wire_map (dict): map of qubits/clbits to position
```

Comment:
Oh, you managed to get this working?  I'm pleased to see this - it looks generally more robust with the new idea of scopes.